### ISSUE TITLE
Cassandra 3.11.4 doesn't startup with AdoptOpenJDK build 25.222-b10

### ISSUE THREAD

**Issue by NitishBangera** [AdoptOpenJDK/openjdk-build#1219]
* **Browser**:

My Java version is the following

openjdk version "1.8.0_222"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_222-b10)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)

I am in the process of evaluating AdoptOpenJdk so that my company moves away from Oracle's Jdk. All my java apps have been working fine but Cassandra 3.11.4 has been giving me issues during startup. 

Issues
1. The jna-4.2.2.jar which comes bundled doesn't creates the dll. When its upgraded to jna-4.5.2.jar then the jna works fine.
2. On startup, the JVM crashes with the following error

```
# JRE version: OpenJDK Runtime Environment (8.0_222-b10) (build 1.8.0_222-b10)
# Java VM: OpenJDK 64-Bit Server VM (25.222-b10 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# C  [sigar-amd64-winnt.dll+0x14ed4]
```

I do understand that its outside the JVM but as soon as I shift to the following Oracle's jdk, cassandra starts up pretty fine without issues and I am able to do all the functionalities.
java version "1.8.0_221"
Java(TM) SE Runtime Environment (build 1.8.0_221-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.221-b11, mixed mode)

I am on hold for the migration due to this issue.

**Comment by karianna** [comment]
Are you able to try the Upstream OpenJDK? adoptopenjdk.net/upstream.html

Let's see if this is a broad OpenJDK issue or just Adopt's

**Comment by NitishBangera** [comment]
Hello Martijn,

Based on your suggestion, I did try the following Upstream OpenJdk
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (build 1.8.0_222-10)
OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode)

To be very specific, I set the JAVA_HOME within the start script so that cassandra uses the appropriate jdk and validated it from the cassandra logs as well. Cassandra started up pretty fine and worked pretty well. So it seems that its mainly an issue with AdoptOpenJdk only.

Thanks,
Nitish

**Comment by karianna** [comment]
Thanks that'll help a little.

@ali-ince Can I get you to look at the DLL etc versions between the two variants and see where we might differ?

**Comment by NitishBangera** [comment]
Hello @karianna ,

Are there any updates for this issue as it's halting my migration efforts from Oracle Jdk to AdoptJdk?

Thanks,
Nitish

**Comment by ali-ince** [comment]
Hi @NitishBangera,

Could you please try the resolution mentioned at https://github.com/AdoptOpenJDK/openjdk-build/issues/1126#issuecomment-512574015? Please consider the architecture of your operating system while picking the version of redistributable to install.

Thanks.

**Comment by NitishBangera** [comment]
Hello @ali-ince,

Thank you for the update. I will be trying your suggestion on windows 10 pro as well as the server os which is Centos 7 and updating this thread. 

Thanks,
Nitish

**Comment by NitishBangera** [comment]
Apologies for the delay as I haven't been able to test the suggestion. I will be updating soon regarding it.

**Comment by amanda-lee-ansys** [comment]
Hi 
I have the same issue here. I am working on migration from Oracle Jdk to AdoptJdk and wanna get rid of all the VS2010 dependencies. Therefore, the solution @ali-ince mentioned does not work for me.

Java version:
adoptOpenJDK 8u222

Cassandra:
tried 3.11.2 and 3.11.4, but both failed.

**Comment by NitishBangera** [comment]
Hello @ali-ince, @karianna 

I would like to echo what @amanda-lee-ansys has mentioned as well. Even I do not want to depend on VS2010 dependencies. Can you please share some other suggestions to solve this issue?

**Comment by cindypodski** [comment]
@karianna @ali-ince - any suggestions for the VS2010 dependencies problem we are encountering?  VS2010 extended support ends in July 2020.

**Comment by NitishBangera** [comment]
@ali-ince @karianna Can you please share some updates?

**Comment by sxa555** [comment]
I would expect that we will be moving up to VS2017 for the JDK8 builds at some point int he next couple of releases.

@ali-ince @karianna anything to add?
